.github/workflows/ruff_formater.yml

```diff
@@ -0,0 +1,22 @@
+name: Ruff formater
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install bleak
```

Comment:
We can skip this if we just want to run the linter

Reply:
Yes, I will remove these for ruff actions.